libmscore/layout.cpp

```diff
@@ -2579,6 +2577,9 @@ std::pair<qreal, qreal> extendedStemLenWithTwoNoteTremolo(Tremolo* tremolo, qrea
 #if 0
       // cross-staff & beam between staves: extend both stems by the same length
       else if (tremolo->crossStaffBeamBetween()) {
+            const qreal sw = tremolo->score()->styleS(Sid::tremoloStrokeWidth).val();
+            const qreal td = tremolo->score()->styleS(Sid::tremoloDistance).val();
+
```

Comment:
This newline seems like it shouldn't be here.

Reply:
separates variables from the code using them. Is done that way in about a million other places

Reply:
Hhmmm. OK not here though, I see now

Reply:
fixed